app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -169,6 +169,7 @@ def rethrow_as_a_miq_error(ovirt_sdk_4_error)
     end
 
     def handle_credentials_verification_error(err)
+      _log.log_backtrace(err)
```

Comment:
This is debug, right?

Reply:
`log_backtrace` is `WARN` I believe

Reply:
oh I didn't recognize that this method was precisely for errors

Reply:
Hm something doesn't seem right about this though:
`[----] E, [2021-12-01T16:57:07.158658 #1699986:91f0] ERROR -- evm: [MiqException::MiqEVMLoginError]: undefined method `async_exec' for nil:NilClass  Method:[block (2 levels) in <class:LogProxy>]`

That's not a backtrace...

Reply:
I'm going to take this out since it isn't directly related to the check_connect_metrics fix and I need to see why it isn't logging the backtrace properly.